espresso-server/app/src/test/java/io/appium/espressoserver/test/http/RouteDefinitionTest.java

```diff
@@ -0,0 +1,65 @@
+package io.appium.espressoserver.test.http;
+
+import fi.iki.elonen.NanoHTTPD;
+import io.appium.espressoserver.lib.handlers.RequestHandler;
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.AppiumParams;
+import io.appium.espressoserver.lib.http.RouteDefinition;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class RouteDefinitionTest {
+    private final RequestHandler<AppiumParams, Void> dumbHandler = new RequestHandler<AppiumParams, Void>() {
```

Comment:
It doesn't work to omit generics when instantiating an anonymous abstract class, for some reason